src/fiatExchanges/FiatExchangeAmount.tsx

```diff
@@ -295,6 +292,37 @@ function FiatExchangeAmount({ route }: Props) {
   )
 }
 
+interface FiatExchangeHeaderProps {
+  title: string | JSX.Element
+  currency: CiCoCurrency
+  displayCrypto: boolean
+}
+
+function FiatExchangeBalanceHeader({
+  title,
+  displayCrypto,
+  currency = CiCoCurrency.CELO,
+}: FiatExchangeHeaderProps) {
+  const tokenInfo = useTokenInfoBySymbol(currency)
+  const subTitle =
+    tokenInfo?.balance != undefined ? (
+      <Trans i18nKey="balanceAvailable">
+        <TokenDisplay
+          amount={tokenInfo.balance}
+          tokenAddress={tokenInfo.address}
+          showLocalAmount={!displayCrypto}
+          hideSign={false}
+          style={styles.headerSubTitle}
+        />
+      </Trans>
+    ) : (
+      // TODO: a null balance doesn't necessarily mean it's loading
```

Comment:
No, this is copy/pasted from when this component was in navigator/Headers.tsx. Its a 2 year old comment and I think its meant to explain the ternary operator above. 

It honestly does not make sense to me, but maybe it will be helpful if we have a bug here someday so I suggest we keep it in

Reply:
sidestepped now by not using this implementation